tests/test_homepage.py

```diff
@@ -16,32 +16,27 @@ class TestHomepage(BaseTest):
 
     @pytest.mark.moztrap([3385, 3386])
     @pytest.mark.nondestructive
-    def test_that_user_can_login_and_logout(self, mozwebqa):
+    def test_that_user_can_login_and_logout(self, mozwebqa, variables):
```

Comment:
Yes, I think I came up with that for later patches on other projects. I'll bring it here too.